DOC-1746-backup-s3-privatelink (#269)
tigergraph/server-docs: modules/backup-and-restore/pages/configurations.adoc

The following is a list of configurations available for backup and restore.

> 
> == C

Author: ```suggestion
```

|`+nan+`

|System.Backup.S3.BucketName |Name of the S3 bucket to store backup files.
Required if backup is to be stored on S3.|null

> |System.Backup.S3.Endpoint |Storage URL (fully qualified) to use instead of default endpoint for S3 client, typically used when operating in a private network or with a non-AWS S3-compatible endpoint. |null

Author: ```suggestion
|System.Backup.S3.Endpoint | Endpoint to use instead of the S3 default endpoint. Must be a fully qualified URL.
Typically used when operating in a private network or with a non-AWS S3-compatible endpoint. | null
```
